yarn.lock

```diff
@@ -11570,6 +11605,11 @@ buffer@^6.0.3:
     base64-js "^1.3.1"
     ieee754 "^1.2.1"
 
+buffers@~0.1.1:
+  version "0.1.1"
+  resolved "https://registry.yarnpkg.com/buffers/-/buffers-0.1.1.tgz#b24579c3bed4d6d396aeee6d9a8ae7f5482ab7bb"
+  integrity sha512-9q/rDEGSb/Qsvv2qvzIzdluL5k7AaJOTrw23z9reQthrbF7is4CtlT0DXyO1oei2DCp4uojjzQ7igaSHp1kAEQ==
+
```

Comment:
@elastic/kibana-operations With 4.0.0 geckodriver module was r[e-written with completely different dependencies](https://github.com/webdriverio-community/node-geckodriver/commit/04a92cd187e341019e77ee544c4550c349be7a4e) and one of them has no any licence specified: it was last published 12y ago https://www.npmjs.com/package/buffers

Licence check on CI failed, how do we usually address such situation?